style.css

```diff
@@ -0,0 +1,565 @@
+@media only screen and (min-width: 768px) {
```

Comment:
- [ ] Although your desktop version addition to your portfolio is impressive :+1:, it is crucial to also ensure that the mobile design is included without relying solely on media queries. This is because your portfolio must be accessible and user-friendly for mobile users as well. Unfortunately, without the mobile design, your portfolio may not be functioning optimally on smaller screens.


https://user-images.githubusercontent.com/105117832/236304702-efe22ac9-c108-46c5-8083-f3b1206d4bc7.mp4